rules/gb-model.smk

```diff
@@ -49,10 +52,25 @@ rule manual_region_merger:
     output:
         merged_shapes=resources("merged_shapes.geojson"),
     log:
-        logs("manual_region_merger.log")
+        logs("manual_region_merger.log"),
     resources:
         mem_mb=1000,
     conda:
         "../envs/environment.yaml"
     script:
         "../scripts/gb-model/manual_region_merger.py"
+
+
+# Rule to retrieve generation unit unavailability data from ENTSO-E
+rule retrieve_unavailability_data:
+    output:
+        planned_unavailability=resources("planned_unavailability.csv"),
+        forced_unavailability=resources("forced_unavailability.csv"),
```

Comment:
add `generator` to the file names since we will have another stream of data for transmission lines (#41 )